tests/sentry/tasks/test_weekly_reports.py

```diff
@@ -281,6 +285,52 @@ def test_organization_project_issue_substatus_summaries(self):
         assert project_ctx.regression_substatus_count == 0
         assert project_ctx.total_substatus_count == 2
 
+    @freeze_time(before_now(days=2).replace(hour=0, minute=0, second=0, microsecond=0))
+    def test_organization_project_issue_status(self):
```

Comment:
this is the new no. 1 flakiest test at sentry -- can you fix it or disable it please?

https://sentry.sentry.io/issues/5084595643/?project=2423079&query=is%3Aunresolved&referrer=issue-stream&sort=freq&statsPeriod=24h&stream_index=0